src/filters/pages.js

```diff
@@ -31,10 +31,26 @@ params:
   page:
     type: 11ty page object
 */
-const isCurrent = (page) =>
-  page.data.end === 'ongoing' ||
-  !page.data.end ||
-  getDate(page.data.end) >= now;
+// eslint-disable-next-line consistent-return
+const isCurrent = (page) => {
```

Comment:
Prettier and eslint rules may be conflicting here, so had to disable the `consistent-return` rule  for this check for undefined pages.

Reply:
Not exactly; the linter wants a function to either explicitly `return` from every branch, or not explicitly `return` at all. It guards against implicit `undefined` return values -- if you want that you have to do `return undefined`. So in this case it was complaining because there was no explicit return value if !page.

https://eslint.org/docs/rules/consistent-return